README.md

```diff
@@ -33,14 +33,58 @@ Deployment of a third-party solution requires a PR for a FluxCD deployment submi
 
 ## Pre-requisite (Linux/MacOS)
 
-This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Please follow the below steps for installing these pre-requisites :
+This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Flux expects to access a source repository and this repository, the api to sync the source code, must be reachable. Please follow the below steps for installing these pre-requisites.
+
+If you are staring from scratch, i.e. you do not have a kubernetes cluster sitting around waiting for you, you can use a kubernetes distribution such as [k3s](https://k3s.io/) to set up a kubernetes cluster on the system you want to use for development/testing.
```

Comment:
Setting up cluster with K3s is not required. We expect users to have a k8s cluster running.

Reply:
Well that was one of my problems. There were implicit requirements in the document and I had to work backwards what was really expected. Which is why I set out on the journey to help make things easier for others. Also the original had "Please follow the below steps for installing these pre-requisites" where pre-requisites are defined in the sentence prior but the steps only talked about installing flux, which is only 1 of the pre-requists. I just don't think the potential contributor should be left hanging w.r.t. the cluster. Setting up k8s is not necessarily for the faint of heart.

Anyway, I can turn this around if that is more to your liking, something like this?

"A running kubernetes cluster is required. If you do not have access to a running cluster you can use a kubernetes distribution like..... or you can setup a cluster in EKS (https://aws.amazon.com/eks/) to enable you to test your changes."